src/HotChocolate/Core/src/Types.NodaTime/HotChocolate.Types.NodaTime.csproj

```diff
@@ -0,0 +1,41 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Nullable>enable</Nullable>
+    <WarningsAsErrors>$(WarningsAsErrors);nullable</WarningsAsErrors>
```

Comment:
WarningsAsErrors for `nullable` wasn't enable any higher, so did it here. Assume it might be too hard higher up.

Reply:
yeah ... we are still fixing nullable warnings in other areas ... but in this lib it us actually the right thing to do.